pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -236,6 +236,32 @@ type PersistentVolumeController struct {
 	// For testing only: hook to intercept CSI driver name <=> Intree plugin name mapping
 	// Not used when set to nil
 	csiNameFromIntreeNameHook func(pluginName string) (string, error)
+
+	// For CSI provisioning/deletion related metrics recording
```

Comment:
This description will need some update to remove CSI as it covers both in-tree and external CSI workflow